lib/pages/home/home_footer.dart

```diff
@@ -25,7 +26,10 @@ class HomeFooter extends StatelessWidget {
             Expanded(
               child: Text(
                 survey.description,
-                style: Theme.of(context).textTheme.bodyText1,
+                style: Theme.of(context)
+                    .textTheme
+                    .bodyText1!
```

Comment:
please avoid using not null `!` here 😸 btw, `style` accept nullable object so replace `!` with `?` is preferred

Reply:
@manh-t Fixed by https://github.com/luongvo/flutter-survey/pull/64/commits/093cef48f30c9c070566cb5ec419282d71ac9265